homeassistant/components/home_connect/light.py

```diff
@@ -194,7 +194,7 @@ async def async_update(self) -> None:
 
         else:
             brightness = self.device.appliance.status.get(self._brightness_key, {})
-            if brightness is None:
+            if not brightness:
                 self._brightness = None
             else:
                 self._brightness = ceil((brightness.get(ATTR_VALUE) - 10) * 255 / 90)
```

Comment:
```suggestion
                self._brightness = ceil((brightness[ATTR_VALUE] - 10) * 255 / 90)
```

Reply:
I think this changes the coding style of the file (which always uses get instead of the [] operator) so I guess the author/maintainer of the integration wouldn't like it.

Reply:
Just a note for completion: I would also prefer the [] operator but I'm just trying to keep the patch simple and I'm used to not changing the coding style of other developers when submitting PRs.

Reply:
This has nothing to do with that. The line above you changed already ensures it is in the dict, so it can be referenced directly (without making an unneeded get() method call).

@antlarr Could you apply the suggestion by emontnemery above?

../Frenck

Reply:
There are many cases of `.get(ATTR_VALUE)` in that file (and even more in other files in that component). Do you want me to change all of them to use `[ATTR_VALUE]` or only that one?